### PR TITLE
feat: Added landing page

### PR DESCRIPTION
**Pull request by kjais1720**
- Index.html & index.css for landing page
- Docs.js & docs.css as skeleton structure
screenshot

### CODE REVIEW

File: Docs/docs.css [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+.copy-btn{
+    position: absolute;
+    right: 30px;
+    top: 30px;
+    font-size: 2rem;
+    color: var(--tr-white);
+}
+
+.copy-btn:hover{
+    color: #008080;
```
Comment (nakulsharma15) [inline Docs/docs.css:159]: You can use variable for this color rather than using it directly

File: Docs/docs.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            mainSidebar.classList.remove('active')
+            mainSidebar.parentElement.classList.remove('active');
+        }
+    })
+
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
+else throw err;
+}
```
Comment (nakulsharma15) [inline Docs/docs.js:33]: Code can be indented for better readability

File: Docs/docs.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+            document.querySelector('.toggle-variant-btn.active').classList.remove('active'); //remove active from current active btn
+            btn.classList.add('active');
+        })
+    })    
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
+else throw err;
+}
```
Comment (nakulsharma15) [inline Docs/docs.js:67]: catch block Code can be indented for better readability

File: Docs/docs.css [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    display: none;
+}
+
+.tabs-btn-list .tr-icon-label-btn{
+    justify-content: center;
+}
+
+.copy-btn{
+    position: absolute;
+    right: 30px;
```
Comment (shubhamkumar648) [inline Docs/docs.css:152]: you could use relative unit for more responsive

**Review by shubhamkumar648 — APPROVED** [review]
everything looks good! suggestion:- you could also use rem

**Review by nakulsharma15 — COMMENTED** [review]
Suggested some minor changes.

File: Docs/docs.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+        buttonBottomBorder.style.transform = `translateX(${moveBorderBy * 100}%)`;
+
+        currentActiveTab = targetTab;
+        currentActiveTabBtn = e.target;
+    }
+})
+
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
```
Comment (Sameony) [inline Docs/docs.js]: although its syntactically correct, adding the curly braces after every if/else is a good practice and provides better readability

File: Docs/docs.js (hunk 3 above)
Comment (kjais1720) [reply Docs/docs.js]: Fixed in the latest commit

**Review by Sameony — COMMENTED** [review]
Suggesting minor improvement

File: Docs/docs.js (hunk 2 above)
Comment (kjais1720) [reply Docs/docs.js]: fixed in the latest commit

File: Docs/docs.css (hunk 1 above)
Comment (kjais1720) [reply Docs/docs.css]: fixed in the latest commit

File: Docs/docs.css (hunk 4 above)
Comment (kjais1720) [reply Docs/docs.css]: fixed in the latest commit

File: Docs/docs.js (hunk 2 above)
Comment (shubhamkumar648) [reply Docs/docs.js]: Perfect

**Review by shubhamkumar648 — APPROVED** [review]
Look good to merger!

**Review by nakulsharma15 — APPROVED** [review]

**Review by Sameony — APPROVED** [review]

**Pull request merged**